kubernetes/testing-scripts/test_file_transfers.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash -e
+
+default_size_of_file=1000000000
+default_number_of_files=5
+
+server_address="$1"
+size_of_file="${2:-$default_size_of_file}"
+number_of_files="${3:-$default_number_of_files}"
+
+
+function file_generator {
+
+    length="$1"
+
+    seq 1000000000000000000000000000000000000000000000000000 | head --bytes "$length"
+
+}
+
+
+iterator=1
+while ((iterator<="$number_of_files")); do
+    start_upload_time="$(date +%s)"
+    file_generator "$size_of_file" |                                \
+        curl                                                        \
+            --request POST                                          \
+            --connect-timeout 10                                    \
+            --include                                               \
+            --header "Concent-Upload-Path: test-file-"$iterator""   \
+            --data-binary @-                                        \
+            "http://"$server_address"/upload/"
```

Comment:
This should be `"http://$server_address/upload/"`

Bash will evaluate a variable inside the double quotes just fine. And leaving it outside the quotes defeates the purpose since it won't let you use spaces inside the variable.